### DISCUSSION TITLE
Release 1.1.1 - Pro Tools support

### DISCUSSION THREAD

**Discussion by apohl79** [apohl79/audiogridder#429] [General]
Hi all,

I'm happy to announce that AudioGridder is available as AAX plugin now and runs natively in Pro Tools!

[Downloads](https://audiogridder.com/download/)

Cheers,
Andreas

**Comment by Billi8** [comment]
This is great thanks !!
A good feature might be keeping or syncing window focus with daw. 
Just play/stop could be good as well

**Reply by apohl79** [reply]
Can you please be a little more specific?

**Comment by Billi8** [comment]
Hi thanks for your reply :)
What I meant was , when focusing on a plug-in GUI (that is hosted by AG) if
I want to stop or start playing again , I need to press inside protools to
shift window focus.
Instead of being able to continue adjusting the plug-in and just press
“space” for play/stop.
Thanks !


On Sat, 13 Mar 2021 at 22:57 Andreas Pohl ***@***.***> wrote:

> Can you please be a little more specific?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/429#discussioncomment-479028>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHXP2PEKVX6KOCC3CBRFP4TTDPGS7ANCNFSM4ZB3WE4Q>
> .
>
-- 
*Bill Tsur - Sound Design.*

BillTsur@Gmail.Com

054-7216789

**Reply by apohl79** [reply]
ok interesting. i need to play with it in pro tools more. ag is forwarding the space key to the host app, so in other daw's it lets you play/pause without changing the focus to the daw window.

**Comment by Billi8** [comment]
Thanks.
I’m running the server on local host with screen capture off.
Acoustica Audio plug-ins.

On Sat, 13 Mar 2021 at 23:43 Andreas Pohl ***@***.***> wrote:

> ok interesting. i need to play with it in pro tools more. ag is forwarding
> the space key to the host app, so in other daw's it lets you play/pause
> without changing the focus to the daw window.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/429#discussioncomment-479143>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHXP2PEB2NG7RUA7QNLIGFDTDPL7JANCNFSM4ZB3WE4Q>
> .
>
-- 
*Bill Tsur - Sound Design.*

BillTsur@Gmail.Com

054-7216789

**Reply by apohl79** [reply]
ahhh ok got it! yeah, that's a bit more tricky.

**Comment by gongwayneming** [comment]
hello I want to illustrate a problem that I encounter in version 1.1.1. After I uninstalling some plugins in Win10 sever and click rescan/wipe rescan.The uninstalled plugin name and its category still remained in my DAW AAX. but in the former version I don't have the problem.

**Reply by apohl79** [reply]
thanks! will have a look.

**Reply by apohl79** [reply]
Can you open a "help" discussion thread and attach your server logs and describe what plugin should disappear? I can't reproduce this. When removing a plugin, it disappears after wipe/rescanning as it should be. I guess you have another version of the plugin somewhere, that survived the uninstallation.

**Comment by gongwayneming** [comment]
hello warm greetings to you !Sorry for my late reply concerning to the previous problem. I have confirmed that I have completed uninstalled the Lx480 from Relab and Alschmitt from Leaping Audio in Windows10 but I still can find them in the column even if I wipe and rescan.&nbsp;



---Original---
From: "Andreas ***@***.***&gt;
Date: Wed, Mar 17, 2021 00:51 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [apohl79/audiogridder] Release 1.1.1 - Pro Tools support (#429)




 
Can you open a "help" discussion thread and attach your server logs and describe what plugin should disappear? I can't reproduce this. When removing a plugin, it disappears after wipe/rescanning as it should be. I guess you have another version of the plugin somewhere, that survived the uninstallation.
